### PR TITLE
Update Kotlin version to 1.9.21

### PR DESCRIPTION
**Pull request by FenstonSingel**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,16 +5,15 @@ group=org.jetbrains.kotlinx
...
 
-defaultKotlinVersion=1.8.21
+defaultKotlinVersion=1.9.20
 dokkaVersion=1.8.20
 serializationVersion=1.5.1
 
 java.mainToolchainVersion=8
 java.modularToolchainVersion=11
 
 kotlin.mpp.stability.nowarn=true
```
Comment (ilya-g) [inline gradle.properties:15]: Probably this one could be removed now

**Review by ilya-g — APPROVED** [review]
Please squash commits before merging.
Also note that its just "Kotlin version", not "Kotlin release version"

**Comment by ilya-g** [comment]
Also it would be better to wait a bit for Kotlin 1.9.21

**Comment by FenstonSingel** [comment]
Force-pushed an updated intermediate state of the branch.
I will get back to this PR to finalize it after 1.9.21's release.

> Please squash commits before merging.

Sure thing. GitHub's "Squash and merge" will do the trick, right?

File: gradle.properties (hunk 1 above)
Comment (FenstonSingel) [reply gradle.properties]: Addressed in ca5725b219549e323bfb62f3fb143c6d6cc98b8f.

**Comment by FenstonSingel** [comment]
Kotlin version changed from 1.9.20 to 1.9.21.
PR should be ready for further review and merging.

**Pull request merged**